app/helpers/evaluations_helper.rb

```diff
@@ -29,24 +29,24 @@ def display_score(assignment)
   end
 
   # individual evaluator score
-  def evaluator_score(assignment)
+  def evaluator_score(assignment, format: :zero)
     score = display_score(assignment)
-    return Score.new(0, "0", "N/A") if score == 'N/A'
+    return Score.new(0, format == :zero ? "0" : "-", "-") if score == 'N/A'
```

Comment:
these score functions have gotten a bit out of hand, I'm going to rework this a bit.

Reply:
submitted #470 to merge into this branch with your approval